bundles/io/org.openhab.io.myopenhab/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,54 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.io.myopenhab.internal
+Ignore-Package: org.openhab.io.myopenhab.internal
+Bundle-ManifestVersion: 2
+Bundle-Name: my.openHAB Service Connection Bundle
+Bundle-SymbolicName: org.openhab.io.myopenhab
+Bundle-Version: 1.7.0.qualifier
+Bundle-Activator: org.openhab.io.myopenhab.internal.MyOpenHABServiceActivator
+Service-Component: OSGI-INF/myopenhab.xml
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Import-Package: org.apache.commons.io;version="2.0.1",
+ org.apache.commons.lang;version="2.6.0",
+ org.eclipse.emf.common.notify,
+ org.eclipse.emf.common.util,
+ org.eclipse.emf.ecore,
+ org.eclipse.emf.ecore.resource,
+ org.eclipse.jetty.client;version="8.1.3",
+ org.eclipse.jetty.io;version="8.1.3",
+ org.eclipse.xtext.resource,
+ org.openhab.config.core,
+ org.openhab.core.events,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.persistence,
+ org.openhab.core.scriptengine,
+ org.openhab.core.service,
+ org.openhab.core.types,
+ org.openhab.model.core,
+ org.openhab.model.items,
+ org.openhab.model.sitemap,
+ org.openhab.ui.items,
+ org.osgi.framework,
+ org.osgi.service.cm;version="1.4.0",
+ org.osgi.service.event;version="1.3.0",
+ org.slf4j;version="1.6.1"
+Export-Package: org.openhab.io.myopenhab
```

Comment:
Yes, it's needed for using my.openHAB methods in rules (that's my understanding of how it works)